content/styles/01-atoms/_forms.css

```diff
@@ -1,3 +1,217 @@
-/*------------------------------------*\
-    #FORMS
-\*------------------------------------*/
+input,
+textarea,
+select,
+.select {
+  font-family: var(--content-font);
+  color: var(--gray);
+  background: var(--white);
+  border: none;
+  width: 100%;
+  font-size: 1.7rem;
+  height: 5rem;
+  border-radius: .5rem;
+  padding: 1.5rem;
+  margin: 0;
+  cursor: pointer;
+}
+
+input:focus,
+textarea:focus,
+select:focus {
+  color: var(--primary-color);
+}
+
+/* LABEL */
+
+label {
+  font-weight: bold;
+  color: var(--gray);
+  text-transform: uppercase;
+  display: inline-block;
+  width: 10rem;
+  margin-bottom: 2rem;
+}
+
+/* TEXTAREA */
+
+textarea {
+  min-height: 9.5rem;
+}
+
+/* CHECKBOX */
+
+input[type=checkbox], 
+input[type=radio] {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  -ms-appearance: none;
+  -o-appearance: none;
+  appearance: none;
+  height: 3rem;
+  width: 3rem;
+  background: var(--shadow-color);
```

Comment:
why not using primary or secondary bg color but shadow color?

Reply:
because `var(--shadow-color)` is used for some *background*, *border* and *box-shadow*